WordPress/build.gradle

```diff
@@ -359,7 +361,7 @@ dependencies {
     androidTestImplementation 'tools.fastlane:screengrab:2.0.0',  {
         exclude group: 'com.android.support.test.uiautomator', module: 'uiautomator-v18'
     }
-    androidTestCompile (name:'cloudtestingscreenshotter_lib', ext:'aar') // Screenshots on Firebase Cloud Testing
+    androidTestImplementation (name:'cloudtestingscreenshotter_lib', ext:'aar') // Screenshots on Firebase Cloud Testing
```

Comment:
I yet have to read about the difference between those two 🤔 

I remember being the one adding that line in this `build.gradle` (as part of my Trial task to implement migrating screenshots to CI) and followed the FTL instructions from [here](https://firebase.google.com/docs/test-lab/android/test-screenshots#add_the_screenshot_library_to_your_project), esp step 5 for this line. Wonder what's the difference between those two variants then 🤔

Reply:
Oh well, it was deprecated over `api` or `implementation`, long discussion this one, in terms of why and how! 😅